gis/text_to_centreline_geometry_scripts/README.md

```diff
@@ -0,0 +1,189 @@
+
+# <center>Text Description to Centreline Geometry Automation</center>
+
```

Comment:
Insert Table of Contents (can be generated automagically using the VS Code Markdown All in One extension